### ISSUE TITLE
NVDA install fails when Windows 10's Controlled Folder Access is enabled

### ISSUE THREAD

**Issue by michaelDCurran** [nvaccess/nvda#7696]
### Steps to reproduce:
In windows 10 Fall Creaters update:
* Enable Controlled Folder Access in windows Defender (Virus and Threat Protection -> Virus and Threat Protection Settings -> Controlled Folder Access).
* Install NVDA either by downloading and running the launcher, or via NVDA's update feature.
* If installing from the launcher, ensure that the "create desktop shortcut" checkbox is checked. It may be called "keep existing desktop shortcut" if it detects a previous install. Either way, make sure it is checked.
* Notice that the install fails near the end.

### Expected behavior:
The install should succeed with no errors.

### Actual behavior:
The install fails, as it cannot create the desktop shortcut in the allUsersDesktop directory, due to Controlled folder Access.

### System configuration:
NVDA version: 2017.3
NVDA Installed or portable: installed

Windows version: 10.0.17017

**Comment by CamLV** [comment]
Add the install .exe to allowed apps. Open Windows Defender Security Center, go to Virus and Threat Protection -> Virus and Threat Protection Settings -> Controlled Folder Access and click on Allow an app through Controlled folder access. Navigate to your Downloads folder and select the installer. Be very careful with this and ensure that the app is from a reliable source and only seeking access to needed folders. Controlled folder access is probably an important addition to Windows security and if I was still an IT manager, I would have it enabled on all my users' computers, even though it would mean I would have to do something like this whenever a new app was deployed. It will mean some additional work for you but it is well worth it, unless you have easy access to cryptocurrency that you don't really need. If so, can I have some?

**Comment by Brian1Gaff** [comment]
As I understood this it was the desktop icon creation that was the issue 
here, so updates one supposes can see if its still there and not corrupted 
or changed and hence not try to overwrite it on an update.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
-----
